packages/inter-protocol/test/vaultFactory/test-prioritizedVaults.js

```diff
@@ -35,107 +38,97 @@ function makeCollector() {
-
-  vaults.addVault(
-    'id-fakeVault1',
-    makeFakeVault('id-fakeVault1', AmountMath.make(brand, 20n)),
+  const vaults = makePrioritizedVaults(store);
+  const fakeVault1 = makeFakeVault(
+    'id1-fakeVaultKit',
+    AmountMath.make(brand, 20n),
   );
-
-  vaults.addVault(
-    'id-fakeVault2',
-    makeFakeVault('id-fakeVault2', AmountMath.make(brand, 80n)),
+  const fakeVault2 = makeFakeVault(
+    'id2-fakeVaultKit',
+    AmountMath.make(brand, 80n),
   );
 
-  const collector = makeCollector();
-  rescheduler.resetCalled();
-  Array.from(vaults.entriesPrioritizedGTE(makeRatio(1n, brand, 10n))).map(
-    collector.lookForRatio,
+  vaults.addVault('id-fakeVault1', fakeVault1);
+  vaults.addVault('id-fakeVault2', fakeVault2);
+
+  const crKey = toVaultKey(
+    // @ts-expect-error cast
+    AmountMath.make(brand, 1n),
+    AmountMath.make(brand, 10n),
+    MINIMAL_VAULT_KEY,
   );
-  t.deepEqual(collector.getPercentages(), [80, 20]);
-  t.falsy(rescheduler.called(), 'second vault did not call reschedule()');
+  const vaultsToLiquidate = vaults.removeVaultsBelow(crKey);
+  const collector = makeCollector();
+  Array.from(vaultsToLiquidate.entries()).map(collector.lookForRatio);
+
+  t.deepEqual(collector.getPercentages(), [80, 20], 'expected vault');
 });
 
 test('update changes ratio', async t => {
   const store = makeScalarBigMapStore('orderedVaultStore');
-  const rescheduler = makeRescheduler();
-  const vaults = makePrioritizedVaults(store, rescheduler.fakeReschedule);
+  const vaults = makePrioritizedVaults(store);
 
   // default collateral of makeFakeVaultKit
   const defaultCollateral = AmountMath.make(brand, 100n);
 
   const fakeVault1InitialDebt = AmountMath.make(brand, 20n);
 
-  const fakeVault1 = makeFakeVault('id-fakeVault1', fakeVault1InitialDebt);
-  vaults.addVault('id-fakeVault1', fakeVault1);
+  const fakeVaultID1 = 'id-fakeVault1';
+  const fakeVault1 = makeFakeVault(fakeVaultID1, fakeVault1InitialDebt);
+  vaults.addVault(fakeVaultID1, fakeVault1);
   t.true(
     vaults.hasVaultByAttributes(
       // @ts-expect-error cast
       fakeVault1InitialDebt,
       defaultCollateral,
-      'id-fakeVault1',
+      fakeVaultID1,
     ),
   );
   t.true(
     vaults.hasVaultByAttributes(
       // @ts-expect-error cast
       fakeVault1InitialDebt,
       defaultCollateral,
-      'id-fakeVault1',
+      fakeVaultID1,
     ),
   );
 
-  vaults.addVault(
-    'id-fakeVault2',
-    makeFakeVault('id-fakeVault2', AmountMath.make(brand, 80n)),
-  );
+  const fakeVaultID2 = 'id-fakeVault2';
+  const fakeVault2 = makeFakeVault(fakeVaultID2, AmountMath.make(brand, 80n));
+  vaults.addVault(fakeVaultID2, fakeVault2);
 
   t.deepEqual(Array.from(Array.from(vaults.entries()).map(([k, _v]) => k)), [
-    'fbff4000000000000:id-fakeVault2',
-    'fc014000000000000:id-fakeVault1',
+    `fbff4000000000000:${fakeVaultID2}`,
+    `fc014000000000000:${fakeVaultID1}`,
```

Comment:
I don't see the benefit of templating here. the rest of the string is bound the to value so changing it will break the same ways.

Reply:
I meant templating vs the status quo of inlining the string literal. We usually use constants so that changing it doesn't require updating a bunch of other values, but in this case the key will need updating anyway so not worth using the constant. Not a blocker.